magefile.go

```diff
@@ -84,6 +94,10 @@ func Verify() error {
 		return err
 	}
 
+	if err := DownloadLicenseData(); err != nil {
```

Comment:
we also should add this to the build / buildstaging parts as well or at least make sure that runs when we invoke any target, maybe in a init func?